producer.go

```diff
@@ -98,8 +97,7 @@ func (p *Producer) consumeDeliveries() {
 				atomic.AddUint64(&stats.producerErr, 1)
 			}
 		} else {
-			p.log.Printf("Error consuming delivery event: Unknown event type (%s)", ev)
-			atomic.AddUint64(&stats.producerErr, 1)
+			p.log.Printf("Ignored event (%s)", ev)
```

Comment:
Since there is a [comprehensive list](https://github.com/edenhill/librdkafka/wiki/FAQ#why-am-i-seeing-receive-failed-disconnected) of the things that could have gone wrong, I think we should evaluate which of this are important and which are not.